index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <p>LazyCoder</p>
+      <img src="./menu.png" alt="" />
+    </header>
+    <section>
```

Comment:
The section should have a proper background image as can be seen in the design
![back](https://user-images.githubusercontent.com/35504149/138149834-423164fe-28cd-45bd-addd-2ac93984af13.png)
.